fs/ops.go

```diff
@@ -112,6 +178,7 @@ func copyDirectory(source, dest string) error {
 			}
 			continue
 		}
+		// TODO: handle symlinks
```

Comment:
Still TODO ?

Reply:
Ya, this was an existing bug I realized wasn't handled. I'd like to fix this in another PR